src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,160 @@
+.contacts {
+  display: grid;
+  grid-template-columns: repeat(2, 130px);
```

Comment:
use unit of measurment fr instead fixed width (problems with the positioning of element)